mobile/src/main/res/menu/log_menu.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/delete_log"
+        android:icon="@drawable/ic_delete_white_24dp"
+        android:title="@string/delete_log"
+        app:showAsAction="always" />
```

Comment:
Isn't this action rather 'clear and update' instead of 'delete'? In that case, a refresh icon would also make a bit more sense to me.

Reply:
IMO a refresh icon would be incorrect as the user thinks it will add new entries, but it removes all entries.

Reply:
Fair enough. MatLog uses a 'close' icon for that purpose ... how about that one?

Reply:
IMO the delete icon is the best choice here.